src/amber/exceptions/page.cr

```diff
@@ -0,0 +1,16 @@
+require "exception_page"
```

Comment:
Same here, we should add this to the templates so the user can modify it or remove it.